tutorials/axios/axios-data.md

```diff
@@ -0,0 +1,5 @@
+When making a standard axios request, the response gives back six properties including `data`, the one we will be discussing. `data` is simply the response that was provided by the server.
```

Comment:
"the response contains 6 properties, including `data`."

Also, let's add a little more detail on what `data` can be. Depending on content-type, data can be a string or an object.